src/ui/spring-pane/index.erb

```diff
@@ -72,7 +72,7 @@
             <div class="input-group-prepend">
               <span class="input-group-text text-monospace">m</span>
             </div>
-            <input id="mass_<%= node_id %>" type="text" placeholder="<%= user_force %>" readonly class="form-control" style="text-align: right;">
+            <input id="mass_<%= node_id %>" type="text" placeholder="<%= user_weight %>" class="form-control" onchange="window.sketchup.user_weight_change(<%= node_id %>, this.value);"  style="text-align: right;">
```

Comment:
I've just checked the usability of the feature: maybe set the user weight as `value` instead of `placeholder`, otherwise it feels strange because the number you enter grays out..